### ISSUE TITLE
[QUESTION]Can this method be used for 2d point cloud registration？

### ISSUE THREAD

**Issue by bene-yan** [MIT-SPARK/TEASER-plusplus#116]
**Have you read the documentation?**

- [x] Yes
- [ ] No - then this issue will be closed.

I am reading your paper, but **I am anxious to know whether this method can be used for 2D point cloud registration/ICP？**

Looking forward to your reply.

**Comment by jingnanshi** [comment]
Yes, but we don't have a dedicated 2D solver. You can try manually setting z=0.

**Comment by bene-yan** [comment]
Thanks a lot! I would try it if i get a chance
